fuse/ipns/ipns_unix.go

```diff
@@ -150,9 +150,6 @@ func (s *Root) Lookup(ctx context.Context, name string) (fs.Node, error) {
 	if segments[0] == "ipfs" {
 		p := strings.Join(resolved.Segments()[1:], "/")
 		return &Link{s.IpfsRoot + "/" + p}, nil
-	} else if segments[0] == "ipns" {
-		p := strings.Join(resolved.Segments()[1:], "/")
-		return &Link{s.IpnsRoot + "/" + p}, nil
```

Comment:
On Thu, May 14, 2015 at 09:34:50PM -0700, Juan Batiz-Benet wrote:
> > @@ -150,9 +150,6 @@ func (s *Root) Lookup(ctx context.Context, name string) (fs.Node, error) {
> >  	if segments[0] == "ipfs" {
> >  		p := strings.Join(resolved.Segments()[1:], "/")
> >  		return &Link{s.IpfsRoot + "/" + p}, nil
> > -	} else if segments[0] == "ipns" {
> > -		p := strings.Join(resolved.Segments()[1:], "/")
> > -		return &Link{s.IpnsRoot + "/" + p}, nil
> 
> curious  why this was removed

From 568bdd22b (namesys: Add recursive resolution, 2015-05-07):

  Now that recursive resolution with the namesys/namesys.go composite
  resolver always gets you to an /ipfs/... path, there's no need for
  the /ipns/ special case in fuse/ipns/ipns_unix.go.

Which sounds like what @whyrusleeping was saying.